src/components/SpaceProposalPage.vue

```diff
@@ -223,6 +229,7 @@ onMounted(() => setMessageVisibility(props.proposal.flagged));
       :space="space"
       :proposal="proposal"
       :selected-choices="selectedChoices"
+      :vote-in-relayer="voteWaitingInRelayer"
```

Comment:
Can you rename `voteInRelayer` and `voteWaitingInRelayer` to `waitingForSigners`, seems way more readable